tdr-admins/tdr-user-administrator.md

```diff
@@ -59,12 +59,15 @@ If a new user needs to be added, then:
     * First Name
     * Last Name
     * Email
-4. In the "Required User Actions" add the following options:  
-  * Configure OTP: *(this will enforce MFA)*
+4. In the "Required User Actions" add **one** the following options:  
```

Comment:
Should this read: ... add one of the following options:

Reply:
Yep, I can't read. I've changed it.